update readme (#18)
gnosis/safe-react-components: README.md

# safe-react-components

![Badge](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg) 
![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/gnosis/safe-react-components?sort=semver)
![GitHub](https://img.shields.io/github/license/gnosis/safe-react-components)

This repository contains a set of React components written in typescript.

These components are being used to build [Safe-multisig](https://github.com/gnosis/safe-react) app.

As Safe-multisig allows to integrate third party applications, Gnosis team recommends to build these apps using this components to get the following benefits:

- Same style as the one used Safe-multisig. Making third-party apps looks native.
- As the space used to render third-party apps is reduced, most of the components will be optimized to render properly in different resolutions.
- Blockchain oriented: Some components will allow you to solve common problems like, inputs for ETH addresses and bigNumbers, Identicon images, etc.

> - Safe time, as you don't have to start from scratch building your safe-app.

Reviewer: typo
```suggestion
- Save time, as you don't have to start from scratch building your safe-app.
```

## How to install

```bash

>    yarn add @gnosis.pm/safe-react-components

Reviewer 2: we should also include npm command

```

## Integration

> This library make use of [material-ui](https://material-ui.com/) as a peerDependency, it means you must install it in your Safe-app. Make sure to provide the same version as the one being use by the current version of this library.

Reviewer: typos
```suggestion
This library makes use of [material-ui](https://material-ui.com/) as a `peerDependency`, it means you must install it in your Safe-app. Make sure to provide the same version as the one being used by the current version of this library.
```

> Once everything is installed, you have to instantiate a [ThemeProvider](https://styled-components.com/docs/api#themeprovider) from [styled-components](https://@gnosis.pm/safe-react-components/).

Reviewer 3: why does `styled-components` link to `https://@gnosis.pm/safe-react-components/`

> This example uses the theme exported by safe-react-components. Here, you could extends this theme to customize it to your needs.

Reviewer: ```suggestion
This example uses the theme exported by safe-react-components. Here, you can extend this theme to customize it to your needs.
```

```js
import { ThemeProvider } from 'styled-components';
import { theme } from '@gnosis.pm/safe-react-components';

import App from './App';

export default () => (
  <ThemeProvider theme={theme}>
    <App />
  </ThemeProvider>
);
```

### Using the same fonts as Safe-multisig

> If you want your app has the same fonts as the one used by Safe-multisig you need to do next.

Reviewer: ```suggestion
If you want your app to have the same fonts as the one used by Safe-multisig you need to do the following.
```

```js
import { createGlobalStyle } from 'styled-components';
import avertaFont from '@gnosis.pm/safe-react-components/dist/fonts/averta-normal.woff2';
import avertaBoldFont from '@gnosis.pm/safe-react-components/dist/fonts/averta-bold.woff2';

const GlobalStyle = createGlobalStyle`
    @font-face {
        font-family: 'Averta';
        src: local('Averta'), local('Averta Bold'),
        url(${avertaFont}) format('woff2'),

>         url(${avertaBoldFont}) format('woff');

Reviewer 2: Please include font-display swap in this example

https://developers.google.com/web/updates/2016/02/font-display

    }
`;

export default GlobalStyle;
```

> And then include it wrapping your application.

Reviewer: Not sure about the _wrapping_ as it's not wrapping it, but at the same `App` level.
